java-backend/src/main/java/media/jambox/Track.java

```diff
@@ -1,29 +1,39 @@
 package media.jambox;
 
 import com.wrapper.spotify.SpotifyApi;
+import com.wrapper.spotify.exceptions.SpotifyWebApiException;
 import com.wrapper.spotify.model_objects.specification.ArtistSimplified;
 import com.wrapper.spotify.model_objects.specification.Image;
 import com.wrapper.spotify.requests.data.tracks.GetTrackRequest;
+import java.io.IOException;
+import java.util.Arrays;
 
 public class Track
+    implements Comparable<Track>
 {
-    private transient String id;
+    private final transient String id;
     private transient String name;
     private transient int durationMS;
     private transient String albumName;
     private transient String[] albumImages;
     private transient String[] artistNames;
-    private int score;
+    private transient int score;
```

Comment:
why is this transient again?

Reply:
"Member variables need to be marked as transient, static, or have accessor methods in the class. Marking variables as transient is the safest and easiest modification. Accessor methods should follow the Java naming conventions, i.e.if you have a variable foo, you should provide getFoo and setFoo methods."